src/components/Popover/index.js

```diff
@@ -28,15 +30,15 @@ export default class Popover extends React.Component {
    }
 
    componentDidMount() {
-      window.addEventListener('click', this.hide.bind(this), false)
+      window.addEventListener('click', this.hide, false)
    }
 
    componentWillUnmount() {
-      window.removeEventListener('click', this.hide.bind(this), false)
+      window.removeEventListener('click', this.hide, false)
    }
 
    show(event) {
-      event.stopPropagation()
+      if (event) event.stopPropagation()
 
       this.setState({
          top: event.currentTarget.offsetTop,
```

Comment:
Ugh, oops, that was forgotten remains from changing stuff around whilst trying to make this fix. Fixed in 4db4f2b. I didn't increment the version or add a new build just for this since it won't make a difference. It will get built in next time.